build.gradle

```diff
@@ -9,7 +9,7 @@ buildscript {
 }
 
 plugins {
-    id 'org.ajoberstar.github-pages' version '1.3.2'
+    id 'org.ajoberstar.github-pages' version '1.6.0'
 }
 
 def depVersions = [
```

Comment:
i'd better do this as a separate task in separate branch. Better to keep releases as atomic as possible.